app/controllers/api/auth/stories_controller.rb

```diff
@@ -6,19 +6,44 @@ class Api::Auth::StoriesController < Api::StoriesController
 
   api_versions :v1
 
-  filter_resources_by :account_id
+  filter_resources_by :account_id, :series_id
 
   announce_actions :create, :update, :delete, :publish, :unpublish
 
   represent_with Api::Auth::StoryRepresenter
 
+  def sorted(res)
+    res.order('updated_at desc')
+  end
+
   # ALL stories - not just published and visible
   def scoped(relation)
     relation
   end
 
+  def filtered(resources)
+    if filters.include?('noseries')
+      resources.unseries
+    else
+      super
+    end
+  end
+
   def resources_base
     @stories ||= current_user.approved_account_stories
   end
 
+  def create_resource
+    super.tap do |story|
+      story.creator_id = current_user.id
+      story.account_id ||= story.series.try(:account_id)
+      story.account_id ||= current_user.account_id
+      story.account_id ||= current_user.approved_accounts.first.try(:id)
+    end
+  end
+
+  def filters
```

Comment:
Is this worth lifting up to a mixin / superclass? I see it in stories and series controllers, getting to be a regular pattern.